File: containsDuplicate.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        nums.sort()
        print(nums)
        if(len(nums)==1):
            return False
        a = nums[0]
        for i in range(1 , len(nums)):
            if(a==nums[i]):
                return True
            else:
                a = nums[i]
        else:
            return False

File: ContainsDuplicate.java
import java.util.*;
class Solution {
    public boolean containsDuplicate(int[] nums) {
        Arrays.sort(nums);
        int p = 0;
        for (int i = 1; i < nums.length; i++) {
            if (nums[p] == nums[i]) {
                return true;
            }
            p++;
        }
        return false;
    }
}